mathics/builtin/makeboxes.py

```diff
@@ -470,12 +470,17 @@ def eval_postprefix(self, p, expr, h, prec, f, evaluation):
         else:
             return MakeBoxes(expr, f).evaluate(evaluation)
 
+    # FIXME: prec is sometimes not an Integer for a ListExpession.
+    # And this is recent with respect to PredefinedSymbol revision.
```

Comment:
In WMA, precision is measured in *decimal* digits. This is why `$MachinePrecision` is a `MachineReal` value. This is different for example in mpmath, where precision is defined in terms of an `int` amount of *binary* digits. This is why we need to accept floating point numbers.

Reply:
The code  indicates that "prec" stands for (operator) "precedence", not "precision". Operator precedence should be an integer value.  

Something weird happens though when ListExpression get passed are embedded. Probably they just need to set the precedence to a high value so that they never get parenthesis since a list expression comes with its own outside boundary markers.